torchtext/prototype/t5/modules.py

```diff
@@ -557,3 +557,170 @@ def forward(self, hidden_states):
             hidden_states = hidden_states.to(self.weight.dtype)
 
         return self.weight * hidden_states
+
+
+class T5Layer(nn.Module):
```

Comment:
Add comment with link to where this is taken from. Also let's make sure to use permalinks (i.e. something like this https://github.com/huggingface/transformers/blob/8581a798c0a48fca07b29ce2ca2ef55adcae8c7e/src/transformers/models/t5/modeling_t5.py#L239).

Read https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files for more info